TeamCode/src/main/java/org/firstinspires/ftc/teamcode/auto/ColorDetection.java

```diff
@@ -24,6 +24,15 @@ public void runOpMode() {
 
         MecanumDrive robot = new MecanumDrive(hardwareMap);
```

Comment:
Have a more descriptive file name or comment at the top of this file summarizing the auton